taipy_gui_ext_library/front-end/webpack.config.js

```diff
@@ -11,16 +11,13 @@
  * specific language governing permissions and limitations under the License.
  */
 
-// webpack should be in the node_modules directory, install if not.
-const path = require("path");
 const webpack = require("webpack");
+const path = require("path");
 require("dotenv").config();
 
-module.exports = (env, options) => {
-  const TAIPY_GUI_DIR = process.env.TAIPY_GUI_DIR;
-
+module.exports = (_env, options) => {
```

Comment:
Why _env?

Reply:
Because it is not used, and I have a warning in my IDE. No pain.